Multisig/UI/Transactions/TransactionDetails/LoadingTransactionDetailsViewModel.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 import Combine
 
 class LoadingTransactionDetailsViewModel: ObservableObject {
+    let safe = Selection.current().safe!
```

Comment:
this should not be used as the initialized property because the details view model might be reused without destroying, but the selected safe will be changed. Getting selected safe should be happening during the sign() function execution if it is needed there

Reply:
ok